lib/rules/web/ocm_console/cluster_idp.xyaml

```diff
@@ -0,0 +1,680 @@
+# --- the elements and actions about the idp configuration
+check_idp_hint:
+  elements:
+  - selector:
+      xpath: //div[@id='idpHint']/div[@class='pf-c-alert__icon']
+    timeout: 100
```

Comment:
is not the timeout too long?

Reply:
@yasun1  I met failure when I use timeout 50, then succeed when I set timeout 100. It wouldn't take too long time actually~